nitro-protocol/contracts/ForceMove.sol

```diff
@@ -50,7 +50,6 @@ contract ForceMove is IForceMove, StatusManager {
         SignedVariablePart memory candidate,
         Signature memory challengerSig
     ) external virtual override {
-        _requireCorrectChainId(fixedPart.chainId);
```

Comment:
Did you also remove the definition of this function (I think it is unused now?

Reply:
removed in 308c17ff.